enterprise/internal/campaigns/syncer.go

```diff
@@ -82,7 +82,8 @@ func (s *SyncRegistry) Add(extServiceID int64) {
 	case "GITHUB", "BITBUCKETSERVER":
 	// Supported by campaigns
 	default:
-		log15.Warn("Syncer not started for unsupported code host", "kind", service.Kind)
+		log15.Debug("Syncer not started for unsupported code host", "kind", service.Kind)
```

Comment:
Let's call it "Campaigns syncer" to not confuse future self from repos syncer (to not be scared :D):
```suggestion
		log15.Debug("Campaigns syncer not started for unsupported code host", "kind", service.Kind)
```